### ISSUE TITLE
Duplicate Renewables Module + Missing Input

### ISSUE THREAD

**Issue by jankiv** [mostaphaRoudsari/ladybug#388]
There seems to be two panels with slightly different names but the same inputs and outputs. Also the description implies you can select the module type, but I don't see an input for it. I just updated Ladybug, but there were no changes to the duplicate panels or the missing input for module type. So I am wondering if this is a bug on my computer or missing from the latest update? 

![image](https://cloud.githubusercontent.com/assets/25178111/26251786/6b9629d8-3c7c-11e7-87fb-2696c64f2ac4.png)

![image](https://cloud.githubusercontent.com/assets/25178111/26251806/782076b8-3c7c-11e7-90c4-bb9c8af00b60.png)

**Comment by mostaphaRoudsari** [comment]
cc: @stgeorges

**Comment by stgeorges** [comment]
Hi @jankiv ,

Please accept my apology for having to wait this long the get a reply to your question (thank you for @mostaphaRoudsari !). I get an email notification about every new topic on [Ladybug forum](Also the description implies you can select the module type, but I don't see an input for it.), but in here I have to manually check for the new ones, and to be honest, I do not really do that that often.

Before 12th of April 2017, Ladybug Photovoltaics components supported only general crystalline silicon module. This meant that you were not able to pick a particular module made by a particular manufacturer. Up until that date, the "Photovoltaics Module" component has been used to define this general crystalline silicon module.
From that date onward, two new components ("Import Sandia Photovoltaics Module" and "Import CEC Photovoltaics Module") were introduced which enable picking a specific crystalline silicon or thin-film module from a library of more 14000 models. To accommodate for the arrival of these two new components the "Photovoltaics Module" component was simply renamed to "Simplified Photovoltaics Module". You can find more information about all of this, along with the .gh example file in [this topic](http://www.grasshopper3d.com/xn/detail/2985220:Comment:1732360) at Ladybug forum.
So basically you just have to right click on your "Photovoltaics Module" component and choose "Delete User Object" to remove it, as you do not need it anymore.

Also the description implies you can select the module type, but I don't see an input for it.

Can you please point to the exact place where the module type is mentioned?
Thank you, and once again sorry for the late reply.

**Comment by chriswmackey** [comment]
@jankiv ,
It sounds like @stgeorges has addressed your question and so I am going to close out this issue.  Feel free to reopen it if I am mistaken.
